scripts/Interface/ui/sans_isis/beam_centre.py

```diff
@@ -148,10 +147,18 @@ def set_run_button_to_normal(self):
         self.run_button.setEnabled(True)
 
     def enable_update_front(self, enabled):
-        self.update_front_check_box.setChecked(enabled)
+        self.update_front_radio.setChecked(enabled)
 
     def enable_update_rear(self, enabled):
-        self.update_rear_check_box.setChecked(enabled)
+        self.update_rear_radio.setChecked(enabled)
+
+    def set_scale_millimeters(self):
```

Comment:
Might it be worth reducing a bit of duplication here by consolidating these two methods into something like the below?

```
def set_scale(self, is_millimeters=True):
    if is_millimeters:
        scale = "mm"
    else:
        scale = "m"
    self.rear_centre_label.setText(f"Centre Position - Rear ({scale})")
    self.front_centre_label.setText(f"Centre Position - Front ({scale})")
```